### PR TITLE
fix typo which causes option.relativeAssets to fail unless assetsHttp…

### PR DESCRIPTION
**Pull request by jimmyhchan**
…Prefix is also used

see d7ab7efc3f for where this was introduced

Couldn't get the existing tests to fail so working on setting up a reduced test case. TBD

### CODE REVIEW

**Comment by jimmyhchan** [comment]
FYI the runtime error
```TypeError: Cannot set property 'relativeTo' of undefined
    at EyeglassCompiler.handleNewFile
```
is what led me to the assumed typo/fix

**Comment by stefanpenner** [comment]
cc @chriseppstein

**Comment by stefanpenner** [comment]
@jimmyhchan could you include a test, to ensure we don't regress?

**Comment by jimmyhchan** [comment]
Agreed, tests are needed. I got to find some time and I'm trying to get my head around how the current tests work in order to get to a failing test case which exercises this bug.

**Comment by jimmyhchan** [comment]
@stefanpenner @chriseppstein modified the tests to `assert.fail` if the broccoli build fails. Was able to reproduce the bug and confirm my change fixes the bug. Also put a `catch` block on every other builder as well.

File: test/test_eyeglass_plugin.js [hunk 1]
```diff
@@ -101,7 +106,23 @@ describe("EyeglassCompiler", function () {
...
+      relativeAssets: true
+    });
+
+    let builder = new broccoli.Builder(optimizer);
+
+    return build(builder)
+      .then(outputDir => {
+        assertEqualDirs(outputDir, fixtureOutputDir("basicProject"));
+      })
+      .catch(assertFail);
```
Comment (stefanpenner) [inline test/test_eyeglass_plugin.js]: the test should already fail if the promise is rejected, thats how mocha's promise support works.

Comment (jimmyhchan) [reply test/test_eyeglass_plugin.js]: TIL -- i'll update the tests again.

Comment (jimmyhchan) [reply test/test_eyeglass_plugin.js]: Updated! Confirmed the 1 new test fails without the code change and passes with it.

**Comment by stefanpenner** [comment]
released as v4.0.5 🎉

**Pull request merged**